UITests/ToolbarTests.swift

```diff
@@ -0,0 +1,88 @@
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        tester().tapViewWithAccessibilityIdentifier("url")
+        let textView2 = tester().waitForViewWithAccessibilityLabel("Address and Search") as! UITextField
+        XCTAssertEqual(textView2.text, url1, "Text is url")
+
+        let url2 = "\(webRoot)/numberedPage.html?page=2"
+        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url2)\n")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 2")
+
+        tester().tapViewWithAccessibilityLabel("Back")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Open new tab and then go back to previous tab to test navigation buttons.
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        let addTabButton = tester().waitForViewWithAccessibilityLabel("Add Tab") as! UIButton
+        addTabButton.tap()
+        tester().waitForViewWithAccessibilityLabel("Web content")
+
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().tapViewWithAccessibilityLabel("Page 1")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Test to see if all the buttons are enabled then close tab.
+        tester().waitForTappableViewWithAccessibilityLabel("Back")
+        tester().waitForTappableViewWithAccessibilityLabel("Forward")
+        tester().waitForTappableViewWithAccessibilityLabel("Reload")
+        tester().waitForTappableViewWithAccessibilityLabel("Share")
+        tester().waitForTappableViewWithAccessibilityLabel("Bookmark")
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().swipeViewWithAccessibilityLabel("Page 1", inDirection: KIFSwipeDirection.Left)
+
+        // Go Back to other tab to see if all buttons are disabled.
+        tester().tapViewWithAccessibilityLabel("home")
+        tester().waitForViewWithAccessibilityLabel("Web content")
+
+        let back = tester().waitForViewWithAccessibilityLabel("Back") as! UIButton
+        let forward = tester().waitForViewWithAccessibilityLabel("Forward") as! UIButton
+        let reload = tester().waitForViewWithAccessibilityLabel("Reload") as! UIButton
+        let share = tester().waitForViewWithAccessibilityLabel("Share") as! UIButton
+        let bookmark = tester().waitForViewWithAccessibilityLabel("Bookmark") as! UIButton
+        
+        XCTAssertFalse(back.enabled, "Back button should be disabled")
+        XCTAssertFalse(forward.enabled, "Forward button should be disabled")
+        XCTAssertFalse(reload.enabled, "Reload button should be disabled")
+        XCTAssertFalse(share.enabled, "Share button should be disabled")
+        XCTAssertFalse(bookmark.enabled, "Bookmark button should be disabled")
+
+        // Rotates back to portrait.
+        let value2 = UIInterfaceOrientation.Portrait.rawValue
+        UIDevice.currentDevice().setValue(value2, forKey: "orientation")
```

Comment:
I guess there's no guarantee that the test started out in portrait. Rather than explicitly going to portrait, can you store the orientation value in a variable and then use that value here?